vktrace/vktrace_layer/vktrace_lib_pageguard.cpp

```diff
@@ -182,8 +183,8 @@ void* pageguardAllocateMemory(size_t size)
     if (size != 0)
     {
 #if defined(WIN32)
-        pMemory = (PBYTE)VirtualAlloc(nullptr, pageguardGetAdjustedSize(size), MEM_WRITE_WATCH | MEM_RESERVE | MEM_COMMIT,
```

Comment:
Was this an intended change? I know the MEM_WRITE_WATCH came in recently, so maybe it's just that the branch needs to be rebased again?